### DISCUSSION TITLE
ERR_HTTP2_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by clinvest** [livewire/livewire#7027] [1. Help]
Hello,

On OVH host I have this problem with livewire livewire.js:1 Failed to load resource: net::ERR_HTTP2_PROTOCOL_ERROR

I found this solution https://laracasts.com/discuss/channels/livewire/livewire-https-issue?page=1&replyId=908661

but this is not working for V3 as "no publishable resources for tag [livewire:assets]".

Any idea / way to correct this?

Thank you in advance,

**Comment by lrljoe** [comment]
What exactly are you trying to achieve here? 

The docs are quite clear on your options for the assets, you can use auto injection, or include via your bundler.

There's a PR open in relation to publish assets tho.

**Reply by clinvest** [reply]
Thank you Irljoe, I'm just trying to deploy the project on a hosting provider (OVH) and I get this error on my filament projet (using livewire). I don't have this problem on other provider (like Infomaniak).

It seems server related and in the same time I really don't know what to search / how to fix.

**Reply by lrljoe** [reply]
If you use the injection method, does it work?

LW3 has an open bug where publishing assets doesnt work smoothly.

I'd recommend you either use the bundler approach or injection approach for now
